README.md

```diff
@@ -82,11 +82,11 @@ https://github.com/
 
 ## 👥 Authors <a name="authors"></a>
 
-👤 ** Marco Almada **
```

Comment:
- [ ] Congratulations on your fantastic project! It's running smoothly and impressively. I did notice that **there are a few sections missing** in your `README` file. Would you mind adding them? Specifically, it would be great if you could include some additional information in the `Getting Started` section. To make it easier for you, I've provided a `README` template in this [link](https://github.com/microverseinc/readme-template). 

   ![image](https://github.com/jlberbesi/Awesome_books.github.io/assets/107952724/99e183b6-4af1-41b3-890b-8040f130b28c)

   **_Remember that creating a good README file is a general requirement for this project._**
   ![image](https://github.com/jlberbesi/Awesome_books.github.io/assets/107952724/4a429e5f-b732-43a7-ba37-9717ffefdc67)

Reply:
@taldr27 Thanks, Diego! I truly appreciate your clarity in providing code reviews. It's extremely helpful and valuable to receive feedback that is easy to understand. Your efforts are greatly appreciated!